### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by HariPadBharti**

### CODE REVIEW

**Comment by HariPadBharti** [comment]
Updating conda may be helpful as I felt difficulty on recognizing at first.

File: README.md [hunk 1]
```diff
@@ -40,7 +40,7 @@ If you use ``conda``, you can install as:
 ```bash
 conda install -c conda-forge jupyterlab
 ```
-
+Before
```
Comment (jzf2101) [inline README.md]: Not sure if this line is necessary, also "If" is capitalized right after

File: README.md [hunk 2]
```diff
@@ -53,6 +53,11 @@ after installation to enable the JupyterLab server extension:
 ```bash
 jupyter serverextension enable --py jupyterlab --sys-prefix
 ```
+or you may update conda before installing jupyterlab as:
```
Comment (jzf2101) [inline README.md]: This should probably go in the same section as conda and not after pip

**Comment by jzf2101** [comment]
Did you have problems installing JuptyerLab with your version of conda?

**Comment by HariPadBharti** [comment]
@jzf2101  yes, I had conda in virtualenv and I first had to update before installing jupyter lab. I had older version of conda, not its 5.0.0

**Comment by jzf2101** [comment]
Which one PR should we consider since you have 2?

**Comment by HariPadBharti** [comment]
Please update with commit 4a95979cbabf0cf4624c35f94b445fa10a2bcd2f

**Comment by jzf2101** [comment]
Could you close the PR you don't want?

**Comment by HariPadBharti** [comment]
@jzf2101 yes I closed.

File: README.md [hunk 3]
```diff
@@ -41,6 +41,12 @@ If you use ``conda``, you can install as:
 conda install -c conda-forge jupyterlab
 ```
 
+or you may update conda before installing jupyterlab as:
```
Comment (yuvipanda) [inline README.md]: Thank you for the patch! 

To help us understand this more, can you tell us what specific issues you ran into that were solved by updating conda? Updating conda without an explicit reason can cause issues in other packages that are unrelated to lab, so if we put it in a prominent place like the README we should be careful to provide concrete reasoning.

Comment (yuvipanda) [reply README.md]: @jzf2101 do we have a 'troubleshooting install issues' section in this repo or elsewhere?

Comment (jzf2101) [reply README.md]: We should probably have that I don't believe that should probably go here: 

http://jupyterlab.readthedocs.io/en/stable/getting_started/installation.html

Comment (HariPadBharti) [reply README.md]: @yuvipanda as I mentioned before, I ran into error to update conda first on virtualenv while installing jupyterlab and needed to update conda before installing.

Comment (yuvipanda) [reply README.md]: Can you help me understand what you mean by 'update conda first on virtualenv'? Are you using virtualenv to install conda?

Comment (HariPadBharti) [reply README.md]: I have conda in virtualenv and wanted to install lab. I am able to work but while installing, I was unable to do with your installation code as I got error, your current version doesn't support the command etc, I am not very sure about error but after updating conda, I was able to.

Comment (yuvipanda) [reply README.md]: I don't think installing conda inside virtualenv is a common configuration, and I think it might cause more confusion with unrelated package breaking when put here.

Can you provide what error you got?

Comment (HariPadBharti) [reply README.md]: I have written enough information already. I thought it might be helpful. If you don't want to add, thats fine.

Comment (ellisonbg) [reply README.md]: @HariPadBharti thanks for your contribution. A few comments:

* In the future, I think we should our documentation efforts on the main ReadTheDocs - we can have simple installation directions on the README, but they should be abbreviated and point to the main docs on ReadTheDocs.
* In this case, I think updating conda is mostly outside the scope of our documentation. In the main ReadTheDocs, we could add a troubleshooting section at the end of the install page, but I am hesitant to put such a special case thing, right in the middle of the installation all users will see.
